src/nft/components/collection/Sweep.tsx

```diff
@@ -413,7 +424,7 @@ export function useSweepFetcherParams(
       if (market === 'others') {
         return { contractAddress, traits, markets }
       }
-      if (!markets.includes(market)) return { contractAddress: '', traits: [], markets: [] }
```

Comment:
why this change?

Reply:
if its being filtered its already being passed as a parameter when market === `'others'`. This prevents it from happening twice